website/src/components/index.ts

```diff
@@ -0,0 +1 @@
+export { Install } from './Install'
```

Comment:
Rather than implementing this pattern here and now, in what will be a one-off, can we wait to implement it in the reboot?